windows/src/desktop/kmshell/xml/config.js

```diff
@@ -194,12 +210,37 @@ document.addEventListener("DOMContentLoaded", windowResize);
 
   function list_detail(event,n) {
     var k = document.getElementById('list_'+n);
-    $(k).toggleClass('expanded');
+    list_expanded_selected (k);
     k.focus();
     save_state();
     return false;
   }
 
+  function list_expanded_selected (n){
+    if(loading_state) return;
+    var keyboards = $('div.list_item');
+    keyboards.each(function(index) {
+      if(this.id == n.id) {
+        $(this).toggleClass('expanded');
+      }
+      else {
+        $(this).removeClass('expanded');
+      }
+    });
+  }
+
+  function more_detail(n) {
+    var k = document.getElementById(n);
+    var m = document.getElementById(n+'_more');
+    $(k).toggleClass('grid_rows_hide');
+    if ($(k).hasClass('grid_rows_hide')){
+      $(m).attr("src","/app/expand20.png");
+    }
+    else {
+      $(m).attr("src","/app/collapse20.png");
+    }
+  }
```

Comment:
1. Use consts where possible:

```
    const k = document.getElementById(n);
    const m = document.getElementById(n+'_more');
```

2. Can we rename the variables so something meaningful, particularly the input param?

Reply:
Yep good plan. I will go through and make these changes.